src/main/resources/templates/pet/list.html

```diff
@@ -60,15 +66,26 @@
                     </div>
                     <div class="caption">
                         <h3 th:text="${pet.name}"></h3>
-                        <p>
-                        <ul>
-                            <li th:text="${@dateFormatter.formatToDate(pet.birthDate, #locale)}"/>
-                            <li th:text="${pet.breed.name}"/>
-                            <li th:text="${pet.dewormed}? #{pet.dewormed} : #{pet.not.dewormed}"/>
-                            <li th:text="${pet.sterilized}? #{pet.sterilized} : #{pet.not.sterilized}"/>
-                            <li th:text="${pet.vaccinated}? #{pet.vaccinated} : #{pet.not.vaccinated}"/>
-                        </ul>
-                        </p>
+                        <div class="pet-info-container">
+                            <div>
+                                <ul>
+                                    <li th:text="${@dateFormatter.formatToDate(pet.birthDate, #locale)}"/>
+                                    <li th:text="${pet.breed.name}"/>
+                                    <li th:text="${pet.dewormed}? #{pet.dewormed} : #{pet.not.dewormed}"/>
+                                    <li th:text="${pet.sterilized}? #{pet.sterilized} : #{pet.not.sterilized}"/>
+                                    <li th:text="${pet.vaccinated}? #{pet.vaccinated} : #{pet.not.vaccinated}"/>
+                                </ul>
+                            </div>
+                            <table border="1" th:if="${!pet.vaccines.isEmpty()}">
+                                <!-- Loop through the products list -->
```

Comment:
@yashudhlani please add `<th>` headers to the table so that we can fix an accesibility issue detected by SonarCloud, more information here: https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&pullRequest=341&types=BUG&severities=BLOCKER%2CCRITICAL%2CMAJOR%2CMINOR&inNewCodePeriod=true&sinceLeakPeriod=true&id=josdem_vetlog-spring-boot

Reply:
@josdem added the table header. can you add the es properties?

Reply:
> @josdem added the table header. can you add the es properties?

Hi @yashudhlani , since you are sending a PR from a fork I can not add it, however this is the change we should do in the `messages_es.properties`:

```
pet.table.header.vaccines=Vacunas
pet.table.header.status=Estatus
```